kuma/static/styles/components/wiki/customcss.scss

```diff
@@ -578,5 +578,31 @@ pre span.comment {
     display: inline;
 }
 
+/*
+  Zone-related classes kept after the removal of zones.
+  (See https://bugzilla.mozilla.org/show_bug.cgi?id=1469029)
+ */
+
+/*
+  reference-values:
+
+  This gives an extra indent to a dl, when it's embedded inside another dl,
+  which comes up sometimes in the reference pages. The extra indent indicates
+  that the embedded dl is part of an embedded list, not the main list.
+ */
+dd > dl.reference-values {
```

Comment:
This could for example be a general utility class called `bump-padding` and then defined as follows:

```
.bump-left-padding {
    padding-left: @grid-spacing * 2;
}
```
Ref: https://github.com/mozilla/kuma/blob/master/kuma/static/styles/includes/_vars.scss#L162

Reply:
I like your utility class idea, but in order to do that we'd have to simultaneously change a significant number of reference-document wiki pages that are currently using the `reference-values` class. I think we should perhaps leave that for a future task?

Reply:
Great idea, let's do that, thanks @schalkneethling!